boards/arm/nrf5340dk_nrf5340/Kconfig

```
@@ -95,22 +46,11 @@ config DOMAIN_CPUNET_BOARD
 
 endif #  BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS
 
-if BOARD_NRF5340DK_NRF5340_CPUNET
-
-# BT_CTLR depends on BT. When BT is enabled we should default to also
-# enabling the controller.
-config BT_CTLR
-	default y if BT
-
-config BT_ECC
-	default y if BT
-
 config DOMAIN_CPUAPP_BOARD
-	string
+	string "Board used by the CPUAPP domain"
 	default "nrf5340dk_nrf5340_cpuapp" if BOARD_NRF5340DK_NRF5340_CPUNET
+	depends on BOARD_NRF5340DK_NRF5340_CPUNET
```

The condition for the default value is redundant.
```suggestion
	default "nrf5340dk_nrf5340_cpuapp"
	depends on BOARD_NRF5340DK_NRF5340_CPUNET
```

---

removed, good catch